api/room/schema_query.py

```diff
@@ -14,6 +15,36 @@
 from helpers.calendar.credentials import Credentials
 
 
+def resolve_booked_rooms_analytics(*args, most_booked_rooms=True):
+    instance, info, start_date, end_date = args
+    if most_booked_rooms:
+        first_ten, last_ten = 10, None
+    else:
+        first_ten, last_ten = None, -10
+    query = Room.get_query(info)
+    location_id = admin_roles.admin_location_for_analytics_view()
```

Comment:
Why are we checking for and using admin location yet this endpoint should be open for all users?